CHANGELOG.md

```diff
@@ -6,7 +6,25 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
-Nothing unreleased at the moment.
+### Added
+
+- Moving from sync to async runtime (breaking changes):
+  - Support async implementation (eg openai, langchain) 
+  - Performance improvements
+  - Removed patching of different libraries
+- Elements:
+  - Merged LocalImage and RemoteImage to Image
+  - New Avatar element to display avatars in messages
+- AskFileMessage now supports multi file uploads (small breaking change)
```

Comment:
I'm not sure this is a breaking change. It's a new feature but it doesn't break the current usage if I'm correct?

Reply:
Oh I was looking at the signature, not at the response, thanks 👌